recipes/qpbenchmark/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "qpbenchmark" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qpbenchmark-{{ version }}.tar.gz
+  sha256: b222aa7b57242392f16c4f07b87c6fd1412d3593eb88ca7e96d9725db788a026
+
+build:
+  entry_points:
+    - qpbenchmark = qpbenchmark.benchmark:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - flit-core >=2,<4
+    - pandas >=1.4.3
+    - pip
+    - py-cpuinfo >=9.0.0
+    - qpsolvers >=4.2.0
```

Unless the project imports itself at build time, you probably don't need these.

```suggestion
    - pip
```

---

Thank you for checking :smiley: As you anticipated, I had to add those as the project imports itself at build time.